vm/src/vm.rs

```diff
@@ -53,7 +53,7 @@ pub struct VirtualMachine {
     pub frames: RefCell<Vec<FrameRef>>,
     pub wasm_id: Option<String>,
     pub exceptions: RefCell<Vec<PyObjectRef>>,
-    pub frozen: RefCell<HashMap<String, &'static str>>,
+    pub frozen: RefCell<HashMap<&'static str, &'static bytecode::CodeObject>>,
```

Comment:
What is the reason for the static lifetime parameter? Is a static lifetime good practice? Can't we use `HashMap<&str, bytecode::CodeObject>`?